homeassistant/components/point/binary_sensor.py

```diff
@@ -71,14 +71,13 @@ def __init__(self, point_client, device_id, device_name):
         super().__init__(
             point_client,
             device_id,
-            DEVICES[device_name].get("device_class"),
+            DEVICES[device_name].get("device_class", device_name),
```

Comment:
With this change, the "alarm" device (line 25) would set the `_attr_device_class` to `alarm` which is invalid and cannot be accepted.
The device class must be an item of `BinarySensorDeviceClass` or `None`.

Please refactor the code so that only valid device classes are set.

Reply:
Thanks for spotting this, I've refactored the code.